File: Level0/짝수의합.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n) {
    int res = 0;
    
    for(int i = 0; i <= n; i++){
        if((i) % 2 == 0)
            res += i;
        }
    return res;
    }

File: Level0/짝수의 합.js
function solution(n) {
  let res = 0;
  for (let i = 0; i <= n; i++) {
    if (i % 2 === 0) res += i;
  }
  return res;
}

console.log(solution(10));

function solution(n) {
  var answer = 0;

  for (let i = 2; i <= n; i += 2) answer += i;

  return answer;
}
